lib/classes/CLI.js

```diff
@@ -221,6 +221,9 @@ class CLI {
     this.consoleLog(chalk.yellow.underline('Environment Variables'));
     this.consoleLog(chalk.dim('* Set SLS_DEBUG=* to see debugging logs'));
     this.consoleLog(chalk.dim('* Set SLS_WARNING_DISABLE=* to hide warnings from the output'));
+    this.consoleLog(
+      chalk.dim('* Set SLS_DEPRECATION_DISABLE=* to disable deprications if required')
```

Comment:
I would rewrite it as `* Set SLS_DEPRECATION_DISABLE=* to disable deprecation logs`